ProjectFA/Source/ProjectFA/InGameItem/Weapon/Weapon.cpp

```diff
@@ -47,6 +48,12 @@ void AWeapon::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeP
 	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
 
 	DOREPLIFETIME(AWeapon, WeaponInfo);
+	DOREPLIFETIME(AWeapon, WeaponSkeletal);
```

Comment:
이미 소유한 클라이언트에서는 WeaponSkeletal에 대한 정보를 바꿨으니, 리플리케이션할 때는 아래와 같이 조건을 추가해서 소유자에게는 가지 않도록 하면, 조금이라도 트래픽을 줄일 수 있지 않을까 생각이 듭니다.

```suggestion
	DOREPLIFETIME_CONDITION(AWeapon, WeaponSkeletal, COND_SkipOwner);
```

Reply:
실행 해보니 잘 되고 있습니다. `SkipOwner` 잘 알아두겠습니다!